### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [SuperKogito/SuperKogito.github.io#6] [Announcements]
# about

This is the about me page of Ayoub Malek's blog and website

https://superkogito.github.io/about.html

**Comment by LaserHu** [comment]
Hi Ayoub,

I am Changshuo, using your python package SPAFE to do speech feature extraction. It works well on most of features, however, I find that the frame number/ window number does not increase when the speech length increases with the feature CQCCs. I wonder if there are some mistake related to the variable: num_ceps? Anyway, you give a great implement. Please contact me if you want know more or communicate, my email is laser_hu@163.com

Changshuo,
Best regards.

**Comment by heromoga2000** [comment]
Salam Ayoub . could you please contact me eelkriwi@gmail.com ? i really need you please
